examples/basics/batches.ipynb

```diff
@@ -200,52 +181,111 @@
     {
       "metadata": {},
       "source": [
-        "# Batch Manipulation"
+        "### Create a batch"
       ],
       "cell_type": "markdown"
     },
     {
       "metadata": {},
       "source": [
-        "### Create a Batch:\n"
+        "batch = project.create_batch(\n",
+        "  name=\"Demo-First-Batch\", # Each batch in a project must have a unique name\n",
+        "  global_keys=sample, # A list of data rows or data row ids\n",
+        "  priority=5 # priority between 1(Highest) - 5(lowest)\n",
+        ")\n",
+        "# number of data rows in the batch\n",
+        "print(\"Number of data rows in batch: \", batch.size)"
+      ],
+      "cell_type": "code",
+      "outputs": [],
+      "execution_count": null
+    },
+    {
+      "metadata": {},
+      "source": [
+        "### Create multiple batches\n",
+        "The `project.create_batches()` method accepts up to 1 million data rows.  Batches are chunked into groups of 100k if necessary, which is the maximum batch size. This method takes in a list of either data row IDs or global keys (but not both).\n",
```

Comment:
Can we also add data row objects to "This method takes in a list of either data row IDs or global keys..."